themes/src/main/resources/theme/keycloak.v2/account/src/package.json

```diff
@@ -6,31 +6,34 @@
     "build": "snowpack --optimize && npm run check-types && npm run babel && npm run move-web_modules && npm run copy-pf-resources",
     "babel": "babel --source-maps --extensions \".js,.ts,.tsx\" app/ --out-dir ../resources/",
     "babel:watch": "npm run babel -- --watch",
-    "check-types": "tsc --noImplicitAny --strictNullChecks --jsx react -p ./",
```

Comment:
I think we do want this

Reply:
These should be configured in the `tsconfig` seems to me.